ghost/core/core/boot.js

```diff
@@ -88,12 +88,6 @@ async function initCore({ghostServer, config, frontend}) {
     require('./shared/url-utils');
     debug('End: Load urlUtils');
 
-    // Models are the heart of Ghost - this is a syncronous operation
-    debug('Begin: models');
-    const models = require('./server/models');
-    models.init();
-    debug('End: models');
-
```

Comment:
I'm curious about this part.

I believe that if we don't do the require in the boot file here, it'll get offloaded to whatever file requires it first, which will be the settings-service below. 

This will "hide" the work to build the require cache for models, which I think is fairly hefty?

Reply:
Technically yes. I removed it because there was no more "work" to do, ie. I'm removing the init function.

I can leave a models require in, indifferent either way

Reply:
Requiring is heavy/slow though right? So feels right to me to have it wrapped in debugs and done in boot, if that's ok?

Reply:
Fixed!